src/payment/create-payment-strategy-registry.ts

```diff
@@ -429,6 +430,21 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register(PaymentStrategyType.CHECKOUTCOM_GOOGLE_PAY, () =>
+        new CheckoutcomGooglePayPaymentStrategy(
```

Comment:
Is there a reason why we're still using a sub-class? I can see we're using `GooglePayCustomerStrategy` for customer strategies. Is there a reason why we're not using `GooglePayPaymentStrategy` and rely on `GooglePayCheckoutcomInitializer` to provide a different initialisation implementation?

Reply:
@davidchin I changed it to not use a custom strategy, but to change the signature of the parseResponse to a promise, if you could review it.
RequestSender bump is still needed